src/sbvr-api/permissions.coffee

```diff
@@ -124,7 +125,31 @@ exports.setup = (app, sbvrUtils) ->
 		POSTPARSE: ({ req, request }) ->
 			# If the abstract sql query is already generated then adding permissions will do nothing
 			return if request.abstractSqlQuery?
+			if (request.method == 'POST' and request.odataQuery.property?.resource == 'canAccess')
+				request.odataQuery.options = { '$select': { 'properties': [ { 'name': 'id' } ] }, $top: 1 }
+				{ action, method } = request.values
+				if method? and action?
+					throw new MissingOrInvalidArgumentsError()
+				else if method? and methodPermissions[method]?
+					request.permissionType = methodPermissions[method]
+				else if action?
+					request.permissionType = action
+				else
+					throw new MissingOrInvalidArgumentsError()
+				request.resourceName = request.resourceName.slice(0, request.resourceName.length - '#canAccess'.length)
```

Comment:
This would be better as
```coffee
request.resourceName.slice(0, -'#canAccess'.length)
```
to avoid the need for fetching `request.resourceName.length`